rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/100_knn_nested_search.yml

```diff
@@ -134,3 +137,93 @@ setup:
   - match: {hits.hits.0._id: "3"}
   - match: {hits.hits.0.fields.name.0: "rabbit.jpg"}
   - match: {hits.hits.0.inner_hits.nested.hits.hits.0.fields.nested.0.paragraph_id.0: "0"}
+---
```

Comment:
This test shows the current bug around global top-k gathering in DFS. I still need to figure out how to get the global k docs diversified over the parents given the children documents.